README.md

```diff
@@ -17,16 +17,18 @@ Currently we are not aware of use outside of [puppet-katello](https://github.com
 
 ## Setup
 
-### What candlepin affects
+### What this module does
 
-* Installs and configures a Candlepin server including Tomcat
+* Installs and configures a Candlepin server including Tomcat and OpenJDK 11.
 
 ## Usage
 
 
 ## Limitations
 
 * EL7 (RHEL 7 / CentOS 7)
+* Candlepin must be version 3.2 or later
+* Requires Java 11 (OpenJDK by default)
```

Comment:
Just want to point out, this states that it requires Java 11, but the parameter `java_package` lets users violate this.